nanogpt/nanogpt_runner.cpp

```diff
@@ -0,0 +1,63 @@
+#include <cstdint>
+#include <functional>
+#include <memory>
+#include <unordered_map>
+
+#include "basic_tokenizer.h"
+#include "managed_tensor.h"
+
+#include <executorch/extension/module/module.h>
+#include <executorch/extension/evalue_util/print_evalue.h>
+#include <executorch/runtime/core/exec_aten/exec_aten.h>
+#include <executorch/runtime/core/exec_aten/util/dim_order_util.h>
+#include <executorch/runtime/core/exec_aten/util/tensor_util.h>
+
+using namespace std;
+using namespace torch::executor;
+
+using SizesType = exec_aten::SizesType;
+using DimOrderType = exec_aten::DimOrderType;
+using StridesType = exec_aten::StridesType;
+
+const int maxSeqLen = 1024;
+
+Result<vector<int64_t>> generate(unique_ptr<Module>& llm_model, vector<int64_t>& tokens) {
+    ManagedTensor tensor_tokens(tokens.data(), {1, 3}, ScalarType::Long);
```

Comment:
ManagedTensor here is copied from llama runner could be removed. Once we remove it, our example is totally irrelevant with llama runner

Reply:
Now I lean towards keeping the ManagedTensor, since it can hide sophisticated tensor and tensor_impl logic and make our logic cleaner and easier to understand. Also I have make the ManagedTensor as lean as possible, to only meet our requirement and no related with llama runner. @dbort what do you think?